README.md

```diff
@@ -1,35 +1,41 @@
 # ExternalDNS Operator
 
-The ExternalDNS Operator deploys and manages [ExternalDNS](https://github.com/kubernetes-sigs/external-dns), which dynamically manages
-external DNS records in specific DNS Providers for specific Kubernetes resources.
-
-This Operator is in the early stages of implementation. For the time being, please reference the
+The `ExternalDNS` Operator allows you to deploy and manage [ExternalDNS](https://github.com/kubernetes-sigs/external-dns), a cluster-internal DNS server in Kubernetes which makes Kubernetes resources discoverable through public DNS servers. \
+**Note**: This Operator is in the early stages of implementation. For more information, see
 [ExternalDNS Operator OpenShift Enhancement Proposal](https://github.com/openshift/enhancements/pull/786).
 
-## Deploy operator
+## Deploying the `ExternalDNS` Operator
+The following procedure describes how to deploy the `ExternalDNS` Operator for AWS.
 
-### Quick development
-1. Build and push the operator image to a registry:
+### Installing the `ExternalDNS` Operator by building and pushing the Operator image to a registry
+1. Build and push the Operator image to a registry:
    ```sh
    $ export IMG=<registry>/<username>/external-dns-operator:latest
    $ make image-build
    $ make image-push
    ```
-2. Run `make deploy` to deploy the operator
-3. Now you can deploy an instance of ExternalDNS:
-    * Run the following command to create the credentials secret for AWS:
+2. Run the following command to deploy the `ExternalDNS`  Operator:
```

Comment:
Please remove one space between `ExternalDNS` and Operator.

Reply:
fixed